packages/celotool/src/lib/prometheus.ts

```diff
@@ -286,32 +301,41 @@ function getServiceAccountName(clusterName: string, cloudProvider: string) {
     .replace(/[^a-zA-Z0-9]+$/g, '')
 }
 
-export async function installGrafanaIfNotExists() {
+export async function installGrafanaIfNotExists(
+  context?: string,
+  clusterConfig?: BaseClusterConfig
+) {
   const grafanaExists = await outputIncludes(
     `helm list -A`,
     grafanaReleaseName,
     `grafana exists, skipping install`
   )
   if (!grafanaExists) {
     console.info('Installing grafana')
-    await installGrafana()
+    await installGrafana(context, clusterConfig)
   }
 }
 
-async function installGrafana() {
+async function installGrafana(context?: string, clusterConfig?: BaseClusterConfig) {
   await createNamespaceIfNotExists(kubeNamespace)
   return installGenericHelmChart(
     kubeNamespace,
     grafanaReleaseName,
     grafanaHelmChartPath,
-    await grafanaHelmParameters()
+    await grafanaHelmParameters(context, clusterConfig)
   )
 }
 
-async function grafanaHelmParameters() {
-  const k8sClusterName = fetchEnv(envVar.KUBERNETES_CLUSTER_NAME)
+async function grafanaHelmParameters(context?: string, clusterConfig?: BaseClusterConfig) {
+  const [k8sClusterName] = getK8sContextVars(clusterConfig, context)
   const k8sDomainName = fetchEnv(envVar.CLUSTER_DOMAIN_NAME)
+  // Rename baklavastaging -> baklava
+  const grafanaUrl =
+    k8sClusterName !== 'baklavastaging'
+      ? `${k8sClusterName}-grafana.${k8sDomainName}.org`
+      : `baklava-grafana.${k8sDomainName}.org`
   const values = {
```

Comment:
Could most of these values be in the Helm Charts and values files instead of coming from celotool?

Reply:
I've moved the "static" ones. I've kept as celotool computed values just the one that are dependent on the environment. We could refactor the chart to reduce the values passed with celotool, but considering that the grafana chart is unchanged from the original [grafana chart](https://artifacthub.io/packages/helm/grafana/grafana), I find more valuable keeping in this way and trying to customize the deployment just providing values as needed.